otter/test/utils.py

```diff
@@ -697,6 +697,56 @@ def resolve_stubs(eff):
     return eff_resolve_stubs(base_dispatcher, eff)
 
 
+def retry_sequence(expected_retry_intent, performers,
+                   fallback_dispatcher=None):
+    """
+    Return a function of Intent -> a that performs the retried effect
```

Comment:
oh. and this happens to be inaccurate, since the return value of this function is a two-tuple of `(expected_retry_intent, Intent -> a)`.